presto-geospatial/src/main/java/io/prestosql/plugin/geospatial/GeoFunctions.java

```diff
@@ -1368,6 +1369,106 @@ public static double greatCircleDistance(
         return atan2(sqrt(t1 * t1 + t2 * t2), t3) * EARTH_RADIUS_KM;
     }
 
+    @Description("Decodes a polyline to a linestring")
```

Comment:
Let's move these into a separate file. This one is getting crowded and there's no real reason to have them here. See BingTileFunctions and how it's used as an example.